tests/server/server-executeBulkWrite-004.phpt

```diff
@@ -10,13 +10,6 @@ require_once __DIR__ . "/../utils/basic.inc";
 $manager = new MongoDB\Driver\Manager(REPLICASET);
 $server = $manager->selectServer(new MongoDB\Driver\ReadPreference(MongoDB\Driver\ReadPreference::RP_SECONDARY));
 
-$bulk = new MongoDB\Driver\BulkWrite();
-$bulk->insert(array('wc' => 0));
-
-$result = $server->executeBulkWrite(NS, $bulk, new MongoDB\Driver\WriteConcern(0));
-var_dump($result->isAcknowledged());
-var_dump($result->getInsertedCount());
```

Comment:
@ajdavis: This test currently fails with a "not master" error from the server. Were there changes in libmongoc 1.9.0 regarding the execution path for unacknowledged writes, whereby we now do get a server response and possible raise an error?